packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -175,4 +177,13 @@ export default abstract class EasCommand extends Command {
     await this.analytics.flushAsync();
     return super.finally(err);
   }
+
+  protected baseErrorMessage = 'Command failed';
+  protected override catch(err: CommandError): Promise<any> {
+    const cleanMessage = err.message.startsWith('[GraphQL] ')
+      ? err.message.replace('[GraphQL] ', '')
+      : err.message;
+    Log.error(cleanMessage);
```

Comment:
```suggestion
    Log.error(cleanMessage);
    Log.debug(err);
```
To preserve the original stack trace when someone runs with `EXPO_DEBUG=1`

Reply:
@wkozyra95 Added the log to debug

Reply:
`Log.debug(err.message)` will not display stack trace (it would just print message again which does not help here). You need to print actual error object with `Log.debug(err)`

Reply:
or alternatively instead of creating new error bellow just do 
```
err.message = isGraphQL ? this.baseGraphQLErrorMessage : this.baseErrorMessage
throw err
``` 
This aproach wouldn't work when you run `EXPO_DEBUG=1`, but it would when you run `DEBUG=*`(oclif is respecting this env)

Reply:
Ok, gotcha, fill fix soon